### ISSUE TITLE
Search button does not have an accessible name

### ISSUE THREAD

**Issue by jeremy-jameson** [technology-toolbox/techtoolbox-hugo#24]
This issue was reported by [Lighthouse](https://developers.google.com/web/tools/lighthouse):

![image](https://user-images.githubusercontent.com/1385288/142066958-a43a7987-7e2e-4ffa-ae0f-d2bcc28208c1.png)

Reference:

[Buttons do not have an accessible name](https://web.dev/button-name/?utm_source=lighthouse&utm_medium=devtools)
